### ISSUE TITLE
How to run commands of npm and sails

### ISSUE THREAD

**Issue by ravi97008** [anshooarora/extentx#15]
Hi Anshoo,

I wish to thank you a lot first for giving a great UI reporting tool (I implemented ExtentReports to my project).
I have a doubt that below commands are not getting executed from ExtentX folder path.

npm install
sails lift

i am getting the errors like  bot 'npm and sails' are not recognized . 
Could you please clarify the doubt.

Thanks in advance..

**Comment by anshooarora** [comment]
Did you install the required tools?

http://extentreports.relevantcodes.com/extentx/

**Comment by ravi97008** [comment]
Yes, i installed MongoDB and NodeJS. And also i unzipped ExtentX folder








*Thanks & Regards---------------------------Ravi Kumar*

**Comment by anshooarora** [comment]
Were you able to resolve this?

**Comment by ravi97008** [comment]
No, still the same issue exists








*Thanks & Regards---------------------------Ravi Kumar*

**Comment by anshooarora** [comment]
Are you on Windows?  If yes, see here:  http://blog.teamtreehouse.com/install-node-js-npm-windows

You may have to install npm separately.  Once NPM is installed and added to your PATH, you can use it to install node modules.  If you already have NPM available, you can simply add it to your PATH and should be good to go.

**Comment by ravi97008** [comment]
Hi Anshoo Arora,

I had set the path of NPM and i tried to run the command 'npm install' from
extentx folder path, i am getting the below error. Please have a look into
this.

Note: MongoDb is running during running the above command.

[image: Inline image 1]

Thanks,
Ravi Kumar








*Thanks & Regards---------------------------Ravi Kumar*

**Comment by email2vimalraj** [comment]
@ravi97008 : are you behind the corporate network? Then you might have to by pass the proxy.

Google listed me this: https://www.google.co.in/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=installing%20npm%20module%20behind%20proxy

**Comment by ravi97008** [comment]
Hi ViamlRaj,

Thanks for your reply...

I have executed the command which is available in the link and also ran the
command(npm config set registry http://registry.npmjs.org/). but it is
showing the below error.

please let me know if i am setting the configureation wrongly..

[image: Inline image 1]

Thanks,
Ravi
